src/System.Management.Automation/engine/remoting/commands/NewPSSessionConfigurationFile.cs

```diff
@@ -735,7 +735,7 @@ protected override void ProcessRecord()
                 if (_roleDefinitions == null)
                 {
                     result.Append(SessionConfigurationUtils.ConfigFragment(ConfigFileConstants.RoleDefinitions, RemotingErrorIdStrings.DISCRoleDefinitionsComment,
-                        "@{ 'CONTOSO\\SqlAdmins' = @{ RoleCapabilities = 'SqlAdministration' }; 'CONTOSO\\ServerMonitors' = @{ VisibleCmdlets = 'Get-Process' } } ", streamWriter, true));
+                        "@{ 'CONTOSO\\SqlAdmins' = @{ RoleCapabilities = 'SqlAdministration' }; 'CONTOSO\\SqlManaged' = @{ RoleCapabilityFiles = 'C:\\RoleCapability\\SqlManaged.psrc' }; 'CONTOSO\\ServerMonitors' = @{ VisibleCmdlets = 'Get-Process' } } ", streamWriter, true));
```

Comment:
I notice windows specific paths (`C:\\`) are used in the code, and I assume the related functionalities are not supported on unix plats, but what would be the user experience? `New-PSSessionConfigurationFile` is exposed on unix plats, so will it throw a friendly error when running the following?
```
New-PSSessionConfigurationFile -Path $PSSessionConfigFile -RoleDefinitions @{
           Administrators = @{ RoleCapabilityFiles = "$RoleCapDirectory\NoFile.psrc" }
}
```